NEWS.md

```diff
@@ -1,5 +1,6 @@
 # riskassessment (development version)
 
+* Added testthat tests for dbSelect and dbUpdate functions (#295)
```

Comment:
This feels super nitpicky on my part but we have been adding the NEWS comments at the bottom of this section. (i.e. line 17)

Reply:
OK on the above